easybuild/easyconfigs/b/BBMap/BBMap-35.82-foss-2015b-Java-1.8.0_66.eb

```diff
@@ -0,0 +1,40 @@
+import platform
+
+easyblock = 'MakeCp'
+
+name = 'BBMap'
+version = '35.82'
+
+homepage = 'https://sourceforge.net/projects/bbmap/'
+description = """BBMap short read aligner, and other bioinformatic tools."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+java = 'Java'
+javaver = '1.8.0_66'
+versionsuffix = '-%s-%s' % (java, javaver)
+dependencies = [(java, javaver, '', True)]
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%(name)s_%(version)s.tar.gz' ]
+
+prebuildopts = 'cd jni && '
+
+os_type = platform.system()
+if os_type == 'Linux':
+    buildopts = '-f makefile.linux'
+elif os_type == 'Darwin':
+    buildopts = '-f makefile.osx'
```

Comment:
things like this should be avoided in easyconfigs

please check if there's a constant available for this (see --avail-easyconfig-constants); if not, we'll add the one you require here

The if/else should also be avoided, maybe use a dictionary that maps value of constant to the label to use?